sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -1,22 +1,32 @@
 # Release History
 
 ## 5.1.0-beta.7 (Unreleased)
```

Comment:
nit: if you want, you could already update this date to today so u only have to merge once and then queue release